src/Properties/SchemaAggregator.php

```diff
@@ -160,6 +161,27 @@ private function processColumnUpdates(string $tableName, string $argName, array
                     $firstArg = $firstMethodCall->args[0]->value ?? null;
                     $secondArg = $firstMethodCall->args[1]->value ?? null;
 
+                    if ($firstMethodCall->name->name === 'foreignIdFor') {
+                        if ($firstArg instanceof PhpParser\Node\Expr\ClassConstFetch
+                            && $firstArg->class instanceof PhpParser\Node\Name
+                        ) {
+                            $modelClass = $firstArg->class->toCodeString();
+                        } elseif ($firstArg instanceof PhpParser\Node\Scalar\String_) {
+                            $modelClass = $firstArg->value;
+                        } else {
+                            continue;
+                        }
+
+                        $columnName = Str::snake(class_basename($modelClass)).'_id';
+                        if ($secondArg instanceof PhpParser\Node\Scalar\String_) {
+                            $columnName = $secondArg->value;
+                        }
+
+                        $table->setColumn(new SchemaColumn($columnName, 'mixed', $nullable));
```

Comment:
Why `mixed` ? We can do `int` I think. That'd cover more than %80 of the cases. If people are using UUID and have `string` primary keys, they can override this with `@property` annotations over the class.